### PR TITLE
MGMT-13305: update the min version for lvm

### PR DESCRIPTION
**Pull request by subhamkrai**
Signed-off-by: subhamkrai <srai@redhat.com>

## List all the issues related to this PR

- [ ] New Feature 
- [ ] Enhancement 
- [x] Bug fix
- [ ] Tests
- [ ] Documentation
- [ ] CI/CD 

## What environments does this code impact?

- [ ] Automation (CI, tools, etc)
- [ ] Cloud
- [ ] Operator Managed Deployments
- [x] None

## How was this code tested?

- [ ] assisted-test-infra environment
- [ ] dev-scripts environment
- [ ] Reviewer's test appreciated
- [ ] Waiting for CI to do a full test run
- [ ] Manual (Elaborate on how it was tested)
- [x] No tests needed

## Checklist

- [ ] Title and description added to both, commit and PR.
- [ ] Relevant issues have been associated (see [CONTRIBUTING] guide)
- [ ] This change does not require a documentation update (docstring, `docs`, README, etc)
- [ ] Does this change include unit-tests (note that code changes require unit-tests)

## Reviewers Checklist

- Are the title and description (in both PR and commit) meaningful and clear?
- Is there a bug required (and linked) for this change?
- Should this PR be backported?

[Kubernetes community documentation]: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#commit-message-guidelines
[CONTRIBUTING]: https://github.com/openshift/assisted-service/blob/master/CONTRIBUTING.md

### CODE REVIEW

**Comment by gamli75** [comment]
@subhamkrai please change the title to MGMT-13305

**Comment by subhamkrai** [comment]
> @subhamkrai please change the title to [MGMT-13305](https://issues.redhat.com//browse/MGMT-13305)

done

**Review by gamli75 — APPROVED** [review]

File: internal/operators/manager.go [hunk 1]
```diff
@@ -130,6 +131,7 @@ func (mgr *Manager) GenerateManifests(ctx context.Context, cluster *common.Clust
 		operator := mgr.olmOperators[clusterOperator.Name]
 		if operator != nil {
 			openshiftManifests, manifest, err := operator.GenerateManifests(cluster)
+			mgr.log.Infof("THIS IS CUSTOM BUILD OF SUBHAM %s", string(manifest))
```
Comment (osherdp) [inline internal/operators/manager.go:134]: ?

**Comment by mhrivnak** [comment]
@subhamkrai @gamli75 can we please have a follow-up PR that reverts the changes in manager.go? Looks like that wasn't intended to be merged.

File: internal/operators/manager.go (hunk 1 above)
Comment (subhamkrai) [reply internal/operators/manager.go:134]: > @subhamkrai @gamli75 can we please have a follow-up PR that reverts the changes in manager.go? Looks like that wasn't intended to be merged.

oops, I'll remove this in follow-up pr. Sorry

Comment (subhamkrai) [reply internal/operators/manager.go:134]: https://github.com/openshift/assisted-service/pull/4920

**Pull request merged**